jerry-debugger/jerry-client-ws.py

```diff
@@ -728,6 +757,27 @@ def parse_source(debugger, data):
     else:
         logging.debug("No pending breakpoints")
 
+def printSource(self, line_num):
+    last_bp = self.debugger.last_breakpoint_hit
+    lines = last_bp.function.source
+    if last_bp:
```

Comment:
you are using last_bp before this check. Please move `lines = last_bp.function.source` after this check.